packages/amplify-cli-core/API.md

```diff
@@ -1264,6 +1396,100 @@ export enum PluginAPIError {
     E_UNKNOWN = "E_UNKNOWN"
 }
 
+// @public (undocumented)
+export class PluginCollection implements IPluginCollection {
+    // (undocumented)
+    [key: string]: Array<PluginInfo>;
+}
+
+// @public (undocumented)
+export class PluginInfo implements IPluginInfo {
```

Comment:
do we have to export classes if we have interfaces ?

Reply:
Right now, both `amplify-cli` and `-core` rely on the implementation, `PluginCollection`. Once the broader shift is done, I'm not positive that both will need to be exported. For right now though, it would require either breaking `amplify-cli` or duplicating the code to only export the interface.